rbac/internal/views.py

```diff
@@ -498,6 +501,26 @@ def data_migration(request):
     return HttpResponse("Data migration from V1 to V2 are running in a background worker.", status=202)
 
 
+def bootstrap_tenant(request):
+    """View method for bootstrapping a tenant.
+
+    POST /_private/api/utils/bootstrap_tenant/?org_id=12345&acct=98765
+    org_id is required,
+    acct is the account number which is optional
+    """
+    if request.method != "POST":
+        return HttpResponse('Invalid method, only "POST" is allowed.', status=405)
+    logger.info("Running bootstrap tenant.")
+
+    org_id = request.GET.get("org_id")
+    if not org_id:
+        return HttpResponse('Invalid request, must supply the "org_id" query parameter.', status=400)
+    acct = request.GET.get("acct", "")
+    bootstrap_service = V2TenantBootstrapService(OutboxReplicator())
+    bootstrap_service._get_or_bootstrap_tenant(org_id, acct)
```

Comment:
Is it possible to use a different method here? This one is private, so it's a little surprising to be used outside of the class. It may change or be removed and not expect to break this endpoint.

Two options I can think of:

- use `bootstrap_tenant` – this requires a Tenant, but we could do nothing if the Tenant does not already exist, which might make sense. I don't think we want to fake a new org ID.
- if the tenant is not present, you could also optionally use `new_bootstrapped_tenant` – this should do mostly the same thing as get_or_bootstrap but is a public method. I see though this method misses a few things for creating the tenant – needs to set ready flag and tenant name. If we need to be able to create tenants I would fix that method up and use that one. And we also might wan tto require an explicit flag to allow creating a new Tenant from this endpoint to prevent accidents.

WDYT?